src/kv/committable_tx.h

```diff
@@ -395,8 +409,11 @@ namespace kv
       // This is a signature and, if the ledger chunking flag is enabled, we
```

Comment:
The comment needs updating to reflect the fact that we trigger a new ledger chunk on the snapshot too.